src/docs/options.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: 'Options'
+---
+
+Options are a way to store generic system-wide configuration. They serve a similar purpose to configuration files, but they are backed by a database, so it's possible to change them at runtime without a deploy. This makes options well-suited for rates, quotas, and limits.
+
+## Adding New Options
+
+To add a new system option, add it to [`sentry/options/defaults.py`](https://github.com/getsentry/sentry/blob/master/src/sentry/options/defaults.py) by registering it. e.g.,
+
+```python
+register("performance.some-feature-rate", default=0.0)
+```
+
+### Tips
+
+- namespace your option (e.g., `feature-scope.feature-name`)
+- colocate your new option with related existing options
+- use dashes and not underscores
+- always set a default value with `default=`
+- the value of an option can be any pickleable value
+
+<Alert level="warning">
+  Always split up registering the new option and the code that relies on the new option. Register new options in their own pull requests, ahead of the code changes.
+</Alert>
+
+## Using Options
+
+To check the value of an option, import the `options` module and use the `get` method. e.g.,
```

Comment:
Is it worth mentioning that `options.get` is cached already so there's no need to do it again?

Reply:
Hmm, what do you mean? Do what again?

Reply:
To roll your own caching someone thinking they have to do

```
cache.set(some_key, option.get(...))
```

Reply:
Ah, okay! I'll move the caching/database bit from "Setting Options" a little higher!